documentation/proc-pages/usage/examples.md

```diff
@@ -24,24 +49,6 @@ pip install notebook
 
 for just the `notebook` package.
 
-<h2>Notebooks in VS Code</h2>
-
-The easiest way to run notebooks is in VS Code; this has the additional advantage of being able to 
-debug notebooks. Simply open the `.ipynb` file in VS Code to view and run it. You may be required 
-to select a notebook kernel on first run; be sure to select the virtual environment where PROCESS 
-is installed (e.g. `~/process/.venv`).
-
-<h2>Notebooks via the Jupyter server</h2>
-
-The original way of running Jupyter notebooks has been via a web browser and the Jupyter server. To 
-launch the server, navigate to the `examples` directory and run:
-
-```bash
-python -m notebook
-```
-
-A web browser will open and the notebook can be run from there. If you're using WSL, you'll 
-have to `ctrl + click` the link.
 
 <h2>Notebooks in Docker</h2>
```

Comment:
This section is outdated now that we have removed the dockerfiles. Could you remove it for me in this PR too?